nanoserver-sac2016/Dockerfile

```diff
@@ -8,7 +8,9 @@ RUN dism.exe /online /add-package /packagepath:c:\install\Microsoft-NanoServer-I
     dism.exe /online /add-package /packagepath:c:\install\Microsoft-NanoServer-IIS-Package_English_10-0-14393-0.cab & \
     dism.exe /online /add-package /packagepath:c:\install\Microsoft-NanoServer-IIS-Package_base_10-0-14393-0.cab & \
     rd /s /q c:\install & \
-    powershell -command {start-service was; While ((Get-ItemProperty HKLM:\SYSTEM\CurrentControlSet\Services\WAS\Parameters\ -Name NanoSetup -ErrorAction Ignore) -ne $null) {sleep 1}}
+    powershell -NoProfile -Command \
+    start-service was; \
+    While ((Get-ItemProperty HKLM:\SYSTEM\CurrentControlSet\Services\WAS\Parameters\ -Name NanoSetup -ErrorAction Ignore) -ne $null) {sleep 1}
```

Comment:
I'm not really a fan of sleep 1. Do whatever is right thing in idiomatic PowerShell

Reply:
@shirhatti 

Do you mean use
```
Start-Sleep -Milliseconds 1
```

or just don't sleep?

Reply:
I meant don't use sleep, but as best as I can tell that seems to be the way to do it in PowerShell 😕